Maple2.Server.Game/Manager/Config/ConfigManager.cs

```diff
@@ -138,6 +141,14 @@ public class ConfigManager {
         session.Send(AttributePointPacket.Allocation(statAttributes));
     }
 
+    public IList<long> GetFavoriteDesigners() {
+        return favoriteDesigners;
+    }
+
+    public void UpdateFavoriteDesigners(IList<long> designers) {
```

Comment:
`Add`+`Remove` seems more appropriate to me rather than overwriting the entire list on every modification. Then you can declare as `private readonly IList<long> favoriteDesigners`